.github/workflows/main.yaml

```
@@ -24,6 +24,8 @@ jobs:
         python-version: ["3.9", "3.10", "3.11"]
     steps:
       - uses: actions/checkout@v4
+        with:
+          depth: 0 # checkout tags (not done by default)
```

apologies, the correct setting is `fetch-depth` not `depth`:
```suggestion
          fetch-depth: 0 # checkout tags (not done by default)
```